<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>enmpruntes</class>
 <widget class="QDialog" name="enmpruntes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1230</width>
    <height>663</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Empruntes</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>281</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/logo.png);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>10</y>
     <width>361</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Consulter Empruntes</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="table_empruntes">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>301</y>
     <width>1231</width>
     <height>361</height>
    </rect>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>150</number>
   </attribute>
   <column>
    <property name="text">
     <string>ID EMPRUNTE</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ID PERSONNE</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>NOM</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>PRENOM</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ID LIVRE</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>TITRE LIVRE</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>DATE EMPRUNTE</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>DATE RETURN EMPRUNTE</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="bt_recherche">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>200</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#bt_recherche{
color:black;
background-color:#a0a0a0;
border-width:0px;
border-radius:4px;
}
#bt_recherche:hover{

background-color:#27a9e3;

}</string>
   </property>
   <property name="text">
    <string>Rechercher</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_reload">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>130</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#bt_reload{
color:black;
background-color:#a0a0a0;
border-width:0px;
border-radius:4px;
}
#bt_reload:hover{

background-color:#27a9e3;

}</string>
   </property>
   <property name="text">
    <string>Reload</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Criteres de recherche :</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>210</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>ID emprunte</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ID personne</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Nom</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Prenom</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ID livre</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Nom de livre</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Date emprunte</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="tb_search">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>210</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
